crates/turbopack-dev-server/src/lib.rs

```diff
@@ -419,24 +422,8 @@ pub(crate) fn resource_to_data(
     if vary.query.is_some() {
         data.query = Some(Query::default())
     }
-    if let Some(filter) = vary.headers.as_ref() {
-        let mut headers = BTreeMap::new();
-        if let Some(resource_headers) = resource.headers {
-            for (header_name, header_value) in resource_headers {
-                if !filter.contains(header_name.as_str()) {
-                    continue;
-                }
-                match headers.entry(header_name) {
-                    Entry::Vacant(e) => {
-                        e.insert(HeaderValue::SingleString(header_value));
-                    }
-                    Entry::Occupied(mut e) => {
-                        e.get_mut().extend_with_string(header_value);
-                    }
-                }
-            }
-        }
-        data.headers = Some(headers);
+    if vary.headers.is_some() {
+        data.headers = Some(Headers::default())
```

Comment:
Why was this changed? Seems like it's no longer passing the headers from the ResourceIdentifier to data. This is needed for RSC HMR which sends the `rsc: 1` header over web socket.

Reply:
Damn, this was a change I made earlier when extracting `Headers` into its own struct that I forgot to revert.